packages/cypress-cloud/lib/config/config.ts

```diff
@@ -95,5 +96,18 @@ export async function getMergedConfig(params: ValidatedCurrentsParameters) {
 }
 
 function getConfigFilePath(projectRoot: string | null = null) {
-  return [projectRoot ?? process.cwd(), "currents.config.js"];
+  const filename = "currents.config";
+  const extensions = ["js", "cjs", "ejs", "ts"];
+  const filepaths: string[] = [];
+
+  for (const extension of extensions) {
+    const filepath = path.join(projectRoot ?? process.cwd(), `${filename}.${extension}`);
+    if (fs.existsSync(filepath)) {
+      filepaths.push(filepath);
+    } else {
+      console.warn(`${filepath} does not exist.`);
+    }
+  }
+
+  return filepaths;
```

Comment:
Please take a look at [const fsConfig = require(resolvedPath);](https://github.com/currents-dev/cypress-cloud/blob/0599cba23af142b458da8cb690fac90a9b6bff82/packages/cypress-cloud/lib/config/config.ts#L46-L47)

- the return value should be a valid argument to `path.resolve` - an array of a path elements
- the resolved file can be `require`'d as a commonJS module
- 
A good validation can be a test that makes sure we can actually require different extenstions.